### PR TITLE
[WIP] Scootaloos

### PR DESCRIPTION
**Pull request by unid15**
april fools ?

TODO:

* [ ] Life() proc

* [ ] Health and damage systems

* [ ] Fix shit

* [ ] Put description here

:cl:
 * rscadd: Added octopuses

### CODE REVIEW

File: cfg/admin.txt [hunk 1]
```diff
@@ -0,0 +1,23 @@
+unid role=admin
```
Comment (Probe1) [inline cfg/admin.txt:1]: derp

File: code/modules/mob/living/carbon/alien/alien.dm [hunk 2]
```diff
@@ -82,7 +82,7 @@ In all, this is a lot like the monkey code. /N
 	return plasma
 
 /mob/living/carbon/alien/eyecheck()
-	return 2
+	return EYECHECK_FULL_PROTECTION
```
Comment (Probe1) [inline code/modules/mob/living/carbon/alien/alien.dm:85]: 👌

File: code/modules/mob/living/carbon/martian/examine.dm [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+	if (isDead())
+		msg += "<span class='deadsay'>It is limp and unresponsive, with no signs of life.</span>\n"
+	else
+		msg += "<span class='warning'>"
+		if (src.getBruteLoss())
+			if (src.getBruteLoss() < 30)
+				msg += "It has minor bruising.\n"
+			else
+				msg += "<B>It has severe bruising!</B>\n"
```
Comment (Probe1) [inline code/modules/mob/living/carbon/martian/examine.dm:23]: Do we use ```<b>``` tags for bolding text?  I vaguely remember it being some span stuff.

File: code/modules/mob/living/carbon/martian/examine.dm [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+		msg += "</span>"
+
+	var/butchery = "" //More information about butchering status, check out "code/datums/helper_datums/butchering.dm"
+	if(butchering_drops && butchering_drops.len)
+		for(var/datum/butchering_product/B in butchering_drops)
+			butchery = "[butchery][B.desc_modifier(src)]"
+	if(butchery)
+		msg += "<span class='info'>[butchery]</span>"
+
+	msg += "*---------*</span>"
```
Comment (Probe1) [inline code/modules/mob/living/carbon/martian/examine.dm:40]: What the hecks this code doing?

File: vgstation13.dme [hunk 5]
```diff
@@ -2169,7 +2176,7 @@
 #include "interface\web\interface.dms"
 #include "maprendering\maprendering.dm"
 #include "maps\_map.dm"
-#include "maps\tgstation.dm"
+#include "maps\test_tiny.dm"
```
Comment (Probe1) [inline vgstation13.dme:2179]: 💯

**Review by Probe1 — CHANGES_REQUESTED** [review]

**Comment by SonixApache** [comment]
the name is retarded

File: code/modules/mob/living/carbon/martian/examine.dm (hunk 3 above)
Comment (Iamgoofball) [reply code/modules/mob/living/carbon/martian/examine.dm:23]: `<span class = 'danger'>`

**Comment by carshalash** [comment]
Its the name that won the polls.

**Comment by ultraplen** [comment]
Is this an april fools joke or will we actually be getting jellyfish-people ?

**Comment by SonixApache** [comment]
@carshalash that doesn't make it any less retarded

File: cfg/admin.txt (hunk 1 above)
Comment (BobdaBiscuit) [reply cfg/admin.txt:1]: sneaky coders at it again

**Comment by NigglyWiggly** [comment]
_**YES**_


Edit: I still hate the name, if only because it was originally removed from clown names for a reason, and adding it in as something else just as a little jab at Clusterfack kind of defeats the purpose or removing it in the first place

**Comment by Kitfox88** [comment]
gonna need to sprite some metal slug outfits for the costume vendor

File: cfg/admin.txt (hunk 1 above)
Comment (PJB3005) [reply cfg/admin.txt:1]: Won't matter anyways, this file is overriden automatically by the server.

**Comment by Kurfursten** [comment]
Is the intent to ever make this a roundstart race option, or just for adminbus? If the answer is the former, implementing it as a species like every other race rather than as its own mob type is probably(?) preferable. Knowing whether it is or not really shapes the nature of a lot of discussion. If it's just a gimmick race, no problem, but otherwise we might need to talk about how a lack of space protection / inability to walk over shards / inability to wear gloves would shut out quite a few jobs.

I'd also like it if you posted a full list of pros and cons, I believe I saw in their code they don't drop things when slipped, which is great because otherwise it would be totally impossible to play a sec officer or something as a scootaloo (always carrying tasers in hand) -- and also that they have full welding protection.

**Comment by Ephx1** [comment]
> Roundstart race option

Only if we remove Skeletons and Vox.

File: code/modules/mob/living/carbon/martian/examine.dm (hunk 4 above)
Comment (Intigracy) [reply code/modules/mob/living/carbon/martian/examine.dm:40]: It's there on every mob examine if you'd notice ingame, it marks the end of the examine for easy viewing.

**Comment by unid15** [comment]
They're never ever becoming a species until we get an organ rework and an inventory slot rework

**Comment by Intigracy** [comment]
But my aunt's name is Scootaloo

**Comment by Riekland** [comment]
um actually the plural of octupus is octopussyi sweetie xx

**Comment by Kurfursten** [comment]
Cool; since they're not really a 'for real' job consideration doesn't matter

Go nuts!

**Comment by invoid9** [comment]
Is voting for the name still an option? My vote goes to Quagaar, since their head kinda looks like a fucked up roast chicken.

**Comment by Ephx1** [comment]
Quagaar warrior job when?

**Comment by ghost** [comment]
if these arent round start im going to shoot myself irl. other than that the pr looks great and i cannot wait

**Comment by TheFriggenSawbones** [comment]
you're a blessed man unid

**Comment by Intigracy** [comment]
Finish it my man

**Comment by unid15** [comment]
I'm currently taking a break, if any of you guys want to pick this up you're welcome to do so, and I'll gladly try to help if any questions come up

**Pull request closed without merging**